apps/fares/lib/proposed_locations.ex

```diff
@@ -0,0 +1,75 @@
+defmodule Fares.ProposedLocations do
+  @moduledoc """
+    Gets information from the existing ArcGIS data related to proposed sales locations, parses the json and returns it.
+  """
+
+  alias Stops.Stop
+  alias Fares.ProposedLocations.Location
+
+  require Logger
+
+  @base_url "https://services1.arcgis.com/ceiitspzDAHrdGO1/ArcGIS/rest/services/ProposedSalesNetworkSpringOutreach/FeatureServer/0/query?f=json&outFields=*&outSR=4326&returnGeometry=true"
+
+  @spec by_stop_ids([Stop.t()]) :: [Location.t()] | nil
+  def by_stop_ids(stop_ids) do
```

Comment:
Why not do an initial getting and parsing of the entire dataset, and then later just filter those results by the distance? Why involve stop_ids?

Reply:
I just thought getting _all_ the data was a lot, that's why my idea was to grab nearby stops and query the ArcGIS data with that. However, @hannahpurcell thought of an even better query which returns nearby locations by lat/lon! 🎉 so that's what I did instead.